start.py

```diff
@@ -0,0 +1,10 @@
+def main() -> None:
```

Comment:
Please delete this file, all work is supposed to be located in the designated folder.